.eslintrc.json

```diff
@@ -8,6 +8,7 @@
         "node": true
     },
     "rules": {
+        "no-debugger": "error",
```

Comment:
🤔

Reply:
in case I commit with `debugger` statement again